tutorials/iot/graphical-snaps/graphical-snaps-xwayland.md

```diff
@@ -0,0 +1,341 @@
+---
+id: graphical-snaps-xwayland
+summary: A guide to creating graphical snaps for Ubuntu Core
+categories: iot
+status: draft
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+tags: snap, digital-signage, kiosk, device, xwayland
+difficulty: 3
+published: 2018-05-21
+author: Gerry Boland <gerry.boland@canonical.com>
+
+---
+
+# Graphical Snaps for Ubuntu Core: Using Xwayland
+
+## Overview
+duration: 1:00
+
+negative
+: Do not attempt this tutorial unless you are already familiar with the material in [Graphical Snaps for Ubuntu Core](tutorial/graphical-snaps). This tutorial covers *only* the extra material needed when using toolkits that do not support Wayland directly.
+ 
+[Graphical Snaps for Ubuntu Core](tutorial/graphical-snaps) is a guide on how to create graphical snaps for Ubuntu Core with a single GUI application running fullscreen on the display. This addresses situations like:
+* Digital signage
+* Web kiosk
+* Industrial machine User Interface
+
+positive
+: The combination of Snap, the "mir-kiosk" Wayland server and Ubuntu Core ensures reliability and security of any graphical embedded device application.
+
+### What you'll learn
+
+How to create graphical snaps for Ubuntu Core using a toolkit that requires Xwayland to supports Wayland.
+
+### What you'll need
+
+You can follow the steps in this tutorial on any current release of Ubuntu. You'll also need a device (or a VM) with Ubuntu core installed.
+
+## Using Wayland
```

Comment:
This is a verbatim copy of the content from the base tutorial, that you say above people should be already familiar with. Should we instead only mention the differences, and the things that matter to this tutorial?